e2etest/pkg/frr/config/unnumbered.go

```diff
@@ -0,0 +1,97 @@
+// SPDX-License-Identifier:Apache-2.0
+
+package config
+
+import (
+	"bytes"
+	"fmt"
+	"os/exec"
+	"strings"
+
+	"go.universe.tf/e2etest/pkg/executor"
+	corev1 "k8s.io/api/core/v1"
+)
+
+const UnnumberedPeerFRRConfig = `
+frr defaults datacenter
+hostname tor
+no ipv6 forwarding
+log file /tmp/frr.log
+!
+interface eth00
+		ipv6 nd ra-interval 10
+		no ipv6 nd suppress-ra
+exit
+!
+interface eth01
+		ipv6 nd ra-interval 10
+		no ipv6 nd suppress-ra
+exit
+!
+interface eth02
+		ipv6 nd ra-interval 10
+		no ipv6 nd suppress-ra
+exit
+!
+interface lo
+		ip address 200.100.100.1/24
+exit
+!
+router bgp 65004
+		bgp router-id 11.11.11.254
+		neighbor MTLB peer-group
+		neighbor MTLB passive
+		neighbor MTLB remote-as external
+		neighbor MTLB description LEAF-MTLB
+		neighbor MTLB bfd
+		neighbor eth00 interface peer-group MTLB
+		neighbor eth00 description k8s-node
+		!
+		address-family ipv4 unicast
+				redistribute connected
+				neighbor MTLB activate
+		exit-address-family
+		!
+		address-family ipv6 unicast
+				redistribute connected
+				neighbor MTLB activate
+		exit-address-family
+	exit`
+
+func WirePeer(peerName string, n corev1.Node) error {
```

Comment:
I'd say this belongs to the container package (has nothing to do with frr), and be renamed something like `wireContainers`. Also, can you add a comment explaining what it does (+ is there some sort of relation between the containers, or what matters is that there are 2 containers?)

Reply:
done
- it does matter because we hardcode the mac in the node part, which is required to calculate the LLA ipv6 address
- having defined mac/LLA is very helpful for debuging

Reply:
I meant in the same file as the `SetupP2PPeer` (which imo should still be called `CreateP2PPeerFor`). Also should we have a const for that mac/lla if that's the case?